x-pack/solutions/security/plugins/security_solution_serverless/public/navigation/side_navigation.ts

```diff
@@ -42,3 +50,83 @@ export const initSideNavigation = async (services: Services) => {
     dataTestSubj: 'securitySolutionSideNav',
   });
 };
+
+// Stack Management static node definition
+const stackManagementLinks: Array<NodeDefinition<AppDeepLinkId, string, string>> = [
+  {
+    title: i18n.translate('xpack.securitySolutionServerless.navLinks.projectSettings.mngt.data', {
+      defaultMessage: 'Data',
+    }),
+    breadcrumbStatus: 'hidden',
+    children: [
+      { link: 'management:index_management', breadcrumbStatus: 'hidden' },
+      { link: 'management:transform', breadcrumbStatus: 'hidden' },
+      { link: 'management:ingest_pipelines', breadcrumbStatus: 'hidden' },
+      { link: 'management:dataViews', breadcrumbStatus: 'hidden' },
+      { link: 'management:jobsListLink', breadcrumbStatus: 'hidden' },
+      { link: 'management:pipelines', breadcrumbStatus: 'hidden' },
+      { link: 'management:data_quality', breadcrumbStatus: 'hidden' },
+      { link: 'management:data_usage', breadcrumbStatus: 'hidden' },
```

Comment:
This one does not exist. Should we clean it?

Reply:
Sorry about that. It was added in the original PR that this work was pulled out of: https://github.com/elastic/kibana/pull/208632. I will cherry-pick.